export-api-def/USAGE

```diff
@@ -0,0 +1,10 @@
+Description:
+  Generate Swagger API definitions for LoopBack application.
+
+Example:
+
+  slc loopback:export-api-def [--json]
+
+  This will output the api definitions to stdout.
+  Use option --json to display in JSON format, otherwise it defaults to YAML.
+  Output can be redirected to a file by adding > output.yaml
```

Comment:
It would be nice to have `-o outfile` so that users don't have to rely on output redirection. It will also allow us to trigger the file-based format detection which is already implemented by loopback-api-definition.